_data/accessibility-tests/search.yml

```diff
@@ -0,0 +1,175 @@
+title: Search
+component_status: pass
+test_items:
+# General tests
+  - summary: Color is not the only method used to convey the search button's meaning.
+    summary_additional: When using the search button, "search" text or a magnifying glass icon convey the button's meaning.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.1
+  - summary: Search button background and text meet color contrast requirements. 
+    summary_additional: |
+      When you use ANDI color contrast analyzer on the search text button, the contrast between background and lettering is at least 4.5:1.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.3
+  - summary: Search button background and icon meet color contrast requirements.
+    summary_additional: |
+      When you use ANDI color contrast analyzer on the search icon button, the contrast between the background and icon is at least 3:1.
+    test_status: conditional
+    test_type: general
+    version_tested: 3.7.1
+    wcag_criterion: 1.4.11
+  - summary: The width of the search button is at least 44 pixels.
+    summary_additional: |
+      When you select "inspect element" to review the button's size, you can see the total width of the button is at least 44 pixels.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 2.5.5
+  - summary: Search field and button views adjust on mobile.
+    summary_additional: |
+      When you rotate between portrait mode and landscape mode on a mobile device, the search button and field are not restricted to a single view.
```

Comment:
Since the criterion is "Orientation" I definitely think we should make that the focal point of the test. I like your additional summary, what if we just update the main summary?

```suggestion
  - summary: Search component matches device orientation
    summary_additional: |
      On mobile devices and tablets, the search component should work in both portrait and landscape orientation with no loss of functionality.
```

Reply:
Looks good! Going to resolve this comment thread, but I added a comment about the updated text in this [comment thread](https://github.com/uswds/uswds-site/pull/2694#discussion_r1636732628).